src/fideslang/models.py

```diff
@@ -991,6 +991,10 @@ class PrivacyDeclaration(BaseModel):
     features: List[str] = Field(
         default_factory=list, description="The features of processing personal data."
     )
+    flexible_legal_basis_for_processing: bool = Field(
+        description="Whether the legal basis for processing is flexible (i.e. editable) for this declaration.",
```

Comment:
```suggestion
        description="Whether the legal basis for processing is 'flexible' (i.e. can be overridden in a privacy notice) for this declaration.",
```
Slightly better phrasing, but that's about the best we can do here! It's always going to be a mouthful.

Reply:
it's overriden in the notice? sorta curious what that means, but i will go with your suggestion here 👍